assets/sass/protocol/includes/mixins/_typography.scss

```diff
@@ -55,223 +55,68 @@
 //  .intro { @include text-body-lg; }
 
 @mixin text-title-2xl {
-    @include font-size(themes.$title-xl-size);
-    line-height: themes.$title-xl-line-height;
-
-    @media #{tokens.$mq-md} {
-        @include font-size(themes.$title-2xl-size);
-        line-height: themes.$title-2xl-line-height;
-    }
-
-    @supports (--css: variables) {
-        font-size: var(--title-xl-size);
-        line-height: var(--title-xl-line-height);
-
-        @media #{tokens.$mq-md} {
-            font-size: var(--title-2xl-size);
-            line-height: var(--title-2xl-line-height);
-        }
-    }
+    font-size: var(--title-2xl-size);
```

Comment:
Would putting the CSS variable into a Sass variable help with readability and authoring speed? Or would it add to confusion?

It would be SO NICE to just type `--title-2xl-size`.

Reply:
Turns out this is a reserved namespace in Sass.